docs/UserGuide.md

```diff
@@ -76,11 +76,11 @@ Format: `help`
 
 Adds a resident to the housing management system.
 
-Format: `radd n/NAME p/PHONE_NUMBER e/EMAIL y/YEAR`
+Format: `radd n/NAME p/PHONE_NUMBER e/EMAIL y/YEAR r/ROOM`
```

Comment:
It should be optional for radd so can you update it?

Reply:
Yes, its optional. I showed two variations in the use examples. But I'll explicitly put that r/ is optional under the radd explanation

Reply:
Ah, I was referring to adding the brackets i.e. [r/ROOM] for optional tag